packages/element/src/delta.ts

```
@@ -1543,6 +1888,31 @@ export class ElementsDelta implements DeltaContainer<SceneElementsMap> {
+      ElementsDelta.redrawTextBoundingBoxes(tempScene, changedElements);
+
+      // needs ordered nextElements to avoid z-index binding issues
+      ElementsDelta.redrawBoundArrows(tempScene, changedElements);
+    } catch (e) {
+      console.error(`Couldn't redraw elements`, e);
+
+      if (isTestEnv() || isDevEnv()) {
+        throw e;
+      }
+    } finally {
+      return nextElements;
```

Biome (2.5.0)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)